data-reconciliation-app/src/repositories/kv-repository.ts

```diff
@@ -0,0 +1,69 @@
+import * as ccfapp from "@microsoft/ccf-app";
+
+export interface IKeyValueRepository<T> {
+  set(key: string, value: T): T;
+  get(key: string): T;
+  has(key: string): boolean;
+  keys(): string[];
+  values(): T[];
+  get size(): number;
```

Comment:
Why did you need to use `get` here in the interface definition?

Reply:
To export it as property call style "repo.size"